documentation/versioned_docs/version-2.x/API Reference.md

```diff
@@ -9842,8 +9842,10 @@ Represents a measurement of how much geometry can be loaded.
 
 | Name | Type | Description |
 | :------ | :------ | :------ |
+| `geometryDownloadSizeBytes` | `number` | Number of bytes of the geometry that must be downloaded. |
+| `highDetailProximityThreshold` | `number` | Sectors within this distance from the camera will always be loaded in high details. |
+| `maximumNumberOfDrawCalls` | `number` | Estimated maximum number of WebGL draw calls to download geometry for. Draw calls are very important for the framerate. |
 | `maximumRenderCost` | `number` | Maximum render cost. This number can be thought of as triangle count, although the number doesn't match this directly. |
-| `highDetailProximityThreshold` | `number` | For 3D models processed before Q1 2022, sectors within this distance from the camera will always be loaded in high details (deprecated). |
```

Comment:
Is this change supposed to be here? I thought this change was made recently

Reply:
This is from master, so shouldn't be a problem, I think. Change is more than 2 months old. I've just used master version for 2.x version.

Reply:
Hmmm, have we removed this in master?

Reply:
It's not in master, I believe it's only merged into release/3, and will only be in the docs when 3.0 lands

Reply:
Well, It's from master in both version 2.x and Next, that's for sure. You can check here - https://cognitedata.github.io/reveal-docs/docs/API%20Reference  by searching `maximumRenderCost`.

Reply:
It is possible that it's not in release because of wrong merge

Reply:
I'll remove this change...